model/submission.py

```diff
@@ -96,6 +96,17 @@ def create_submission(user, language_type, problem_id):
                 'got': language_type
             },
         )
+    # reset quota if it's a new day
+    if user.last_submit.date() != now.date():
```

Comment:
if student didn't submit any submission?

Reply:
then he can submit?

Reply:
last_submit is a very early day by default

Reply:
ok